### ISSUE TITLE
Catalog get "/meta" should return a dictionary

### ISSUE THREAD

**Issue by jrchudy** [informatics-isi-edu/ermrest#100]
When you get the `/meta` resource for the catalog object, you get an array of objects. These objects are a key and value pair for different permissions, like the following:
```
{
  k: "content_write_user",
  v: <some_id>
}
```

Ideally, this resource should return a dictionary with a list of users for each permission in that dictionary.
```
{
  "content_write_user": [<some_id>, <another_id>],
  "content_read_user: [...],
  ...
}
```

**Comment by karlcz** [comment]
This is implemented and documented in 255ea0f5184d04a47062c23a6411c88b491b1918.   See https://github.com/informatics-isi-edu/ermrest/blob/master/api-doc/rest-catalog.md#access-control-lists-retrieval  and related API docs.  Note, the serialization output has changed for **all** the meta API modes.

- /meta  produces `{ "ACL": ["member", ...], ... }`
- /meta/ACL  produces `["member", ...]`
- /meta/ACL/member produces `"member"`

**Comment by jrchudy** [comment]
I verified that this issue works properly now when getting the catalog `/meta` resource. Closing this issue.
